src/analytics/navigationTracker/navigationTracker.ts

```diff
@@ -127,6 +127,8 @@ export class NavigationTracker extends BaseTracker {
         return "footer";
       } else if (this.isHeaderMenuBarLink(element)) {
         return "header menu bar";
+      } else if (this.isPhaseBannerLink(element)) {
+        return "phase banner";
```

Comment:
Following the latest doc we have, the type needs to be "header menu bar" when the visitor clicks on the phase banner.
Only the section property (which we will not managing for the moment) needs to take the value "phase banner"
more info: https://govukverify.atlassian.net/browse/DFC-149 (section Header Bar Navigation)
But your condition is good. Just need to be associated to the "if" which return the value "header menu bar"